tools/windows-node-installer/test/e2e/azure_test.go

```diff
@@ -154,7 +154,14 @@ func setup() error {
 
 // populateAzureInfo populates the fields present in the azureInfo.
 func populateAzureInfo() error {
-	oc, err := client.GetOpenShift(kubeconfig)
+	var oc *client.OpenShift
+	var err error
+	// Use client created by service account if running in CI environment
+	if os.Getenv("OPENSHIFT_CI") == "true" {
```

Comment:
Did you test this locally with Azure? If not, please do and post the output of the run.

Reply:
Working on this

Reply:
```
[vaishnavi@localhost windows-node-installer]$ go test -run=TestCreateVM ./test/e2e/... -v
=== RUN   TestCreateVM
2020/05/07 15:29:24 Namespace Created
2020/05/07 15:29:24 Role Created
2020/05/07 15:29:24 Cluster Role Created
2020/05/07 15:29:24 Service Account Created
2020/05/07 15:29:24 Cluster Role Binding created
2020/05/07 15:29:24 Role Binding created
2020/05/07 15:29:25 config updated to use service account permissions
2020/05/07 15:31:31 Namespace Created
2020/05/07 15:31:32 config updated to use service account permissions
=== RUN   TestCreateVM/check_if_required_security_rules_are_present
=== RUN   TestCreateVM/check_if_ansible_is_able_to_ping_on_the_WinRmHttps_port
=== RUN   TestCreateVM/check_if_container_logs_port_is_open_in_Windows_firewall
=== RUN   TestCreateVM/check_if_SSH_connection_is_available
2020/05/07 15:31:50 successfully deleted all resources
--- PASS: TestCreateVM (19.01s)
    --- PASS: TestCreateVM/check_if_required_security_rules_are_present (0.81s)
    --- PASS: TestCreateVM/check_if_ansible_is_able_to_ping_on_the_WinRmHttps_port (7.39s)
    --- PASS: TestCreateVM/check_if_container_logs_port_is_open_in_Windows_firewall (8.22s)
    --- PASS: TestCreateVM/check_if_SSH_connection_is_available (0.85s)
PASS
ok  	github.com/openshift/windows-machine-config-bootstrapper/tools/windows-node-installer/test/e2e	19.015s
```

Reply:
Please post this in the main PR page.